lib/api/core/plugins/pluginsManager.js

```diff
@@ -1063,6 +1065,32 @@ class PluginsManager {
             : {}
         };
 
+      // load customs errors configuration file
+      if (!plugin.manifest.raw.errors) {
+        this.kuzzle.log.warn(`[${plugin.manifest.name}] This plugin doesn't have custom errors in the manifest file.`);
```

Comment:
Using error codes should be an optional features, and as such, it shouldn't log a warning.